ivy/functional/frontends/torch/Tensor.py

```diff
@@ -45,6 +45,9 @@ def float(self, memory_format=torch.preserve_format):
     def tan(self, *, out=None):
         return torch_frontend.tan(self.data, out=out)
 
+    def amax(self, dim=None, keepdim=False, *, out=None):
```

Comment:
Here, the implementation seems to be nice however the parameters for `Tensor.amax` are only dim and keepdim the instance method doesn't have an out argument `Tensor.amax(dim=None, keepdim=False) ` refer to documentation here [Tensor.amax](https://pytorch.org/docs/stable/generated/torch.Tensor.amax.html)